Shelley-testnet/Exercise-1.md

```diff
@@ -34,37 +37,48 @@ Welcome to the IOHK Pioneers Group!  We are excited that you have joined us on t
+1. Download, Build and Install the Cardano Node Software using the instructions in the [IOHK Tutorial](https://github.com/input-output-hk/cardano-tutorials/tree/master/node-setup).  You will need to do this repeatedly as the Testnet progresses, so you may want to bookmark the instructions.  We will be updating the software regularly.  You will need to use the correctly tagged version at each stage. Latest tag is 1.13.0 Generally, you will be able to use the version that is tagged “latest” in the GitHub repository.
 
 2. Start the node using the genesis, topology, network and other configuration settings that will be given to you by IOHK. Follow the instructions in the IOHK tutorial.
 
 ```bash
  cardano-node run --config …
 ```
 
-1. Check that your instance of the node is properly connected to the Pioneer testnet and is verifying the blocks that are produced, as described in the IOHK tutorial.
+1. Check that your instance of the node is properly connected to the Testnet and is verifying the blocks that are produced, as described in the IOHK tutorial.
 
   Congratulations!  You now have a working node that is connected to the first-ever public Shelley Cardano network! In our later exercises, we will first use this to submit transactions, and then set up a working stake pool, so that you can produce blocks yourself.
 
-4. Use the node CLI commands to generate an address key, utxo.txt, that will be used to provide you with funds.  Extract the address from your address key key as described in the IOHK tutorial (linked above), and record it in this [Google Doc](https://docs.google.com/spreadsheets/d/1o62S2_24bcZNIpT3FASKHVGHG-dQ4J0vDaVCEsxbVKU/edit?usp=sharing).
+4. Use the node CLI commands to generate an address key, utxo.txt, that will be used to provide you with funds.  
 
    - [Tutorial on making an address](https://github.com/input-output-hk/cardano-tutorials/blob/master/node-setup/address.md)
 
-5. Confirm on [SurveyMonkey](https://www.surveymonkey.co.uk/r/S9N5M5H) that you have successfully completed this exercise, and provide your feedback on any problems that you have encountered.  Once we are happy that all the Pioneers have installed the node and perfomed a test run of the node, and that we have ironed out any issues that you experience, we will progress to the next step: building and submitting transactions.
+5. Request funds from the faucet with
+
+    curl -v -XPOST “https://faucet.ff.dev.cardano.org/send-money/<YOURADDR>?apiKey=CGX2UGexU3CpSW1MMY0HitKVSrAiQuyR”
```

Comment:
Should the faucet URL and apiKey be publicly exposed ?

Reply:
Faucet - yes, API-key: no, I think.